lib/src/expandable_page_view.dart

```diff
@@ -207,7 +207,9 @@ class _ExpandablePageViewState extends State<ExpandablePageView> {
     _sizes = _prepareSizes();
     _pageController = widget.controller ?? PageController();
     _pageController.addListener(_updatePage);
-    _currentPage = _pageController.initialPage.clamp(0, _sizes.length - 1);
+    final sizesLength = _sizes.length;
+    final sizesUpperBounds = sizesLength == 0 ? 0 : sizesLength - 1;
```

Comment:
use `_sizes.isEmpty` instead of checking if length is 0. Irrelevant though, see below